Mage/src/main/java/mage/abilities/effects/common/CreateTokenEffect.java

```diff
@@ -56,7 +57,7 @@ public CreateTokenEffect(Token token, int amount, boolean tapped, boolean attack
 
     public CreateTokenEffect(Token token, DynamicValue amount, boolean tapped, boolean attacking) {
         super(Outcome.PutCreatureInPlay);
-        this.token = token;
+        this.tokens.add(token);
```

Comment:
since we're using get(0) later, should null check token now and throw error on wrong usage